bin/build_exercise_crate.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Compile the 'exercise' crate and put it in the 'bin/' folder
+
+TRACK_ROOT="$(git rev-parse --show-toplevel)"
+
+EXERCISE_CRATE_PATH="$TRACK_ROOT/exercise"
+
+BIN_DIR_PATH="$TRACK_ROOT/bin"
+
+(
+	cd "$EXERCISE_CRATE_PATH"
+
+	echo "Building exercise crate"
+
+	cargo build --release
+
+	RELEASE_PATH="$EXERCISE_CRATE_PATH/target/release/exercise"
+
+	if [ -f "$RELEASE_PATH/exercise.exe" ]; then
```

Comment:
This will never be true. It needs to be written like
```suggestion
	if [ -f "$RELEASE_PATH.exe" ]; then
```